test-suite/dune

```diff
@@ -1,3 +1,13 @@
+; The easiest way to generate a portable absolute path is to use OCaml
+; itself to print it
+(executable
+ (name ocaml_pwd)
+ (modules ocaml_pwd))
+
+(rule
+ (targets libpath.inc)
+ (action (with-stdout-to %{targets} (run ./ocaml_pwd.exe ../../install/default/lib/coq/ ))))
```

Comment:
Does dune always install to `default`?

Reply:
This can be done using a variable I think so indeed it may be a good idea to use the variable.

Reply:
Used `%{context_name}`